source/gen/FlashOWare.Generators/Generators/Enumerations/EnumInterceptorGenerator.cs

```diff
@@ -102,26 +102,26 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
 			document.WriteLine('{');
 			document.Indent++;
 
-			IEnumerable<IGrouping<string, EnumMethodInvocation>> group = source.GroupBy(static value => value.Name);
+			IEnumerable<IGrouping<string, EnumMethodInvocation>> group = source.GroupBy(static value => $"{value.Name}:{value.Enum.Name}");
```

Comment:
I love the fact that this change emits no duplicate code anymore. Really shows by removing the incrementing number suffix.
But utilizing _string-interpolation_ for the key-generation is not ideal, IMO.
_.NET Standard 2.0_ does not have the [DefaultInterpolatedStringHandler Struct](https://learn.microsoft.com/dotnet/api/system.runtime.compilerservices.defaultinterpolatedstringhandler). I'd like to avoid "expensive" string-concatenation here.

When writing this for the first _pre-release_, I actually thought about generating a ``ValueTuple`2``-based key here instead (via C# 7.0 tuple types), and utilizing its equality semantics here. And this Tuple could be given a descriptive name via a _using alias_ now with C# 12.0.

Of course benchmarking is in order to confirm that statement.
I'm currently building a Benchmarking-Library for Roslyn (incremental) Source Generators ... should be ready in 1 week.

Should this comment block this PR?
If you are happy to wait, I'd like to add Benchmarking first in the last week of November, and then get back to this PR and see which variant is really faster.
If you'd rather not delay completing this PR, I could also take a note in #13 to revisit this before publishing the next _pre-release_.

Reply:
I am happy to wait with this PR until the Benchmark is added. In the meantime I will apply your suggestions 😄 

I absolutely agree, that the string-interpolation is not ideal and was more of a quick-shot solution to show what I mean with my issue report.